### ISSUE TITLE
`module describe` appears to be broken from 6.6.3 to 7.1.12

### ISSUE THREAD

**Issue by barrymoo** [TACC/Lmod#206]
I wanted to upgrade our site's Lmod from 6.0.1 to get the `module describe` functionality, however upon upgrade to 7.1.12, I found that `module describe` wasn't working. I downgraded to 6.6.3 and it worked fine. Notes: I added `ls ~/.lmod.d/` because I noticed the functionality doesn't play nicely between versions (i.e. the directory was emptied). Configured with: `configure --prefix=... --with-module-root-path=...`.  Lua Version:

```
[bmooreii@login0b ~]$ lua -v
Lua 5.1.4  Copyright (C) 1994-2008 Lua.org, PUC-Rio
```

From 6.6.3:

```
[bmooreii@login0b ~]$ module --version

Modules based on Lua: Version 6.6.3  2016-11-13 15:27 -07:00
    by Robert McLay mclay@tacc.utexas.edu

[bmooreii@login0b ~]$ ls ~/.lmod.d/
[bmooreii@login0b ~]$ module load python intel intel-mpi boost-compiled
[bmooreii@login0b ~]$ module save boost
Saved current collection of modules to: boost
[bmooreii@login0b ~]$ module describe boost
Collection "boost" contains: 
   1) python    2) intel    3) intel-mpi    4) boost-compiled
```

From 7.1.12:

```
[bmooreii@login0b ~]$ module --version

Modules based on Lua: Version 7.1.12  2016-12-29 20:55 -06:00
    by Robert McLay mclay@tacc.utexas.edu

[bmooreii@login0b ~]$ ls ~/.lmod.d/
[bmooreii@login0b ~]$ module load python intel intel-mpi boost-compiled
[bmooreii@login0b ~]$ module save boost
Saved current collection of modules to: boost
[bmooreii@login0b ~]$ module describe boost
/usr/bin/lua: /ihome/crc/install/lmod/lmod/libexec/cmdfuncs.lua:141: attempt to concatenate upvalue 'system_name' (a boolean value)
stack traceback:
	/ihome/crc/install/lmod/lmod/libexec/cmdfuncs.lua:141: in function 'cmd'
	/ihome/crc/install/lmod/lmod/libexec/lmod:473: in function 'main'
	/ihome/crc/install/lmod/lmod/libexec/lmod:520: in main chunk
	[C]: ?
```

**Comment by rtmclay** [comment]
Thanks for the bug report!  I have released 7.2.1 which fixes the "module describe" problem you pointed out in 7.1.12.  Please let me know if this version fixes this problem for you.

**Comment by barrymoo** [comment]
Works like a charm. Thanks @rtmclay
